### DISCUSSION TITLE
[FEATURE REQUEST} Vectrex compatibility

### DISCUSSION THREAD

**Discussion by joselito75** [OpenStickCommunity/GP2040-CE#589] [Ideas]
Hi, is it possible to make this board compatible with the Vectrex console? 
Vectrex use a DB9 plug same as Sega Genesis/Megadrive and adapters from USB to DB9 already exists
Vectrex original controllers are so hard to find, bad quality and very expensive
I like the idea of a DIY joystick with this board that will be compatible with most of the consoles and computers I own

**Comment by bsstephan** [comment]
Classic console support is in its nascent phase of design. I encourage you to keep an eye open for the mid-term future when this might be more possible. We have bounties (currently, only on the Discord, AFAIK) for a number of major consoles, and it's possible someone will be able to work on this request at some point in the future. (However, we're more focused on consoles that fit in an RJ45 rather than DB9, right now.)

**Comment by TheTrainGoes** [comment]
Hi everyone,

This is general notice that we are closing down the discussion section of the GP2040-CE repo.  

If you would like to discuss anything further please open an issue or join our Discord:
https://discord.com/servers/openstickcommunity-1049366310389289001
